plugin/trino-memsql/src/main/java/io/trino/plugin/memsql/MemSqlClient.java

```diff
@@ -145,6 +146,7 @@
     static final int MEMSQL_TEXT_MAX_LENGTH = 65535;
     static final int MEMSQL_MEDIUMTEXT_MAX_LENGTH = 16777215;
     private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("uuuu-MM-dd");
+    private static final Pattern UNSIGNED_TYPE_REGEX = Pattern.compile("(?i)(?s).*unsigned$");
```

Comment:
why `?s`? I don't expect there to be newlines here.

Reply:
👍 I'll fix that